common/Stress/Azure.Sample.Stress/MultiConstructorTest.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Test.Stress;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.Sample.Stress
+{
+    // Verifies the perf framework handles multiple constructors
+    public class MultiConstructorTest : StressTest<StressOptions, StressMetrics>
```

Comment:
This seems like it should be part of the Test project and not a sample. Rationale: It is specific to a test-only scenario, though I can see how it may be a common enough scenario that demonstrating via sample would be beneficial. You may want to consider either moving it or renaming to remove the Test suffix.